docker-compose.yaml

```diff
@@ -24,7 +24,7 @@ services:
     container_name: postgres-cache
     environment:
       POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: password
+      POSTGRES_PASSWORD: ${DATABASE_PASSWORD:-null}
```

Comment:
In your `.env-example` you define a `CACHE_DATABASE_PASSWORD` but here you use `DATABASE_PASSWORD`. Isn't this meant to be `CACHE_DATABASE_PASSWORD`?